### PR TITLE
Add reusable ScanList component for displaying previous scans

### PR DESCRIPTION
**Pull request by bryandino673**

### CODE REVIEW

**Review by ValantineSuh — APPROVED** [review]
Good!

File: src/components/ScanViewer.tsx
Comment (stephane-segning) [inline src/components/ScanViewer.tsx:1]: This component is not part of the ticket

File: src/components/ScanList.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    {avatarGroup.map((avatar, index) => (
+      <img
+        key={index}
+        src={avatar}
+        alt={`Avatar ${index}`}
+        className="avatar"
+      />
+    ))}
+  </div>
+);
```
Comment (stephane-segning) [inline src/components/ScanList.tsx:24]: Remove this component into another file

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Change name to pascal case. E.g `ScanView` => `scan-view`

**Review by Ngha-Boris — COMMENTED** [review]
Hello @bryandino673 can you consider adding error handling for cases where the data passed to the components is incomplete or malformed. For example, if avatarGroup or rawData is missing or invalid, you might want to handle these cases gracefully to prevent runtime errors.

File: src/components/ScanViewer.tsx
Comment (stephane-segning) [inline src/components/ScanViewer.tsx:1]: Remove this component

File: package.json
Comment (stephane-segning) [inline package.json:1]: Cancel these changes. They're not related to this ticket

File: .husky/pre-commit
Comment (stephane-segning) [inline .husky/pre-commit:1]: Cancel these changes. They're not related to this ticket

**Review by stephane-segning — CHANGES_REQUESTED** [review]

**Review by Ngha-Boris — APPROVED** [review]

File: src/components/ScanViewer.tsx
Comment (stephane-segning) [inline src/components/ScanViewer.tsx:1]: Remove this component

**Review by stephane-segning — CHANGES_REQUESTED** [review]

File: src/components/ScanViewer.tsx
Comment (bryandino673) [reply src/components/ScanViewer.tsx:1]: Understood sir

**Review by stephane-segning — APPROVED** [review]

**Pull request merged**